src/licensedcode/data/licenses/crapl-0.1.yml

```diff
@@ -0,0 +1,8 @@
+key: crapl-0.1
+short_name: crapl-0.1
+name: CRAPL v0 BETA 1
+category: CRAPL
```

Comment:
This is not a category. Please look at other licenses categories. We may need @DennisClark take on this